adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Exceptions/AdalError.cs

```diff
@@ -33,6 +33,11 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
     /// </summary>
     public static class AdalError
     {
+        /// <summary>
+        /// Integrated Windows Auth is only supported for "federated" users
+        /// </summary>
+        public const string IwaNotSupportedForManagedUser = "integrated_windows_auth_not_supported_managed_user";
```

Comment:
>IwaNotSupportedForManagedUser [](start = 28, length = 29)

I know it will be super long, but maybe spell it out...Iwa looks a little weird. Or just don't use Iwa...`IntWindowsAuthNotSupportedForManagedUser` ?

Reply:
Done.